suite-native/labeling/src/selectors.ts

```diff
@@ -0,0 +1,46 @@
+import {
+    WithLabelingState,
+    WithSuiteSyncAndDeviceState,
+    selectAccountLabel as selectAccountLabelLocalFirst,
+    selectIsFeatureSuiteSyncAvailable,
+} from '@suite-common/suite-sync';
+import {
+    AccountsRootState,
+    selectIsPortfolioTrackerDevice,
+    selectAccountLabel as selectReduxAccountLabel,
+} from '@suite-common/wallet-core';
+import type { Account } from '@suite-common/wallet-types';
+import { parseDeviceStaticSessionId } from '@suite-common/wallet-utils';
+
+export type CombinedLabelingState = WithLabelingState &
+    WithSuiteSyncAndDeviceState &
+    AccountsRootState;
+
+export const selectIsLabelingEnabled = (state: WithSuiteSyncAndDeviceState) => {
+    const isSuiteSyncAvailable = selectIsFeatureSuiteSyncAvailable(state);
+    const isPortfolioTracker = selectIsPortfolioTrackerDevice(state);
+
+    return isSuiteSyncAvailable && !isPortfolioTracker;
+};
+
+export const selectAccountLabel = (
+    state: CombinedLabelingState,
+    accountKey?: Account['key'],
```

Comment:
nit, but :pray: 

```suggestion
    accountKey?: AccountKey,
```

Imo much better to use the Type rather then referencing property type from where its used.

Reply:
Fixed